src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -875,6 +881,32 @@ export class BaseEditCourseForm extends React.Component {
               disabled={disabled}
               optional
             />
+            {showSkills
+            && (
+            <Field
+              name="skill_names"
+              component={CourseSkills}
+              label={(
+                <FieldLabel
+                  id="skills.label"
+                  text="Skills"
+                  helpText={(
+                    <div>
+                      <p>
+                        edX partners with Emsi, the labor market data company, to automatically tag your courses with
+                        in-demand skills from their library of 30,000 skills based on the content in your about page.
+                        If you want to experiment with what skills show up, you can edit and submit changes to your
+                        about page description.
+                      </p>
+                    </div>
+                  )}
+                />
+              )}
+              disabled={disabled}
```

Comment:
Shouldn't disabled always be true? It looked like a read only field.

Reply:
Good catch, I have fixed it.